model/job/worker.go

```diff
@@ -114,16 +114,18 @@ func (w *WorkerConfig) Clone() *WorkerConfig {
 func NewWorkerContext(workerID string, job *Job, inst *instance.Instance) *WorkerContext {
 	ctx := context.Background()
 	id := fmt.Sprintf("%s/%s", workerID, job.ID())
-	log := logger.WithDomain(job.Domain).
-		WithField("job_id", job.ID()).
-		WithField("worker_id", workerID).
-		WithNamespace("jobs")
+	entry := logger.WithDomain(job.Domain).WithNamespace("jobs")
 
 	if job.ForwardLogs {
 		hook := realtime.LogHook(job, realtime.GetHub(), consts.Jobs, job.ID())
-		log.AddHook(hook)
+		entry.AddHook(hook)
 	}
 
+	log := entry.
+		WithNamespace(job.WorkerType).
```

Comment:
I don't think we want to override the namespace set a few lines before (`WithNamespace("jobs")`)